tests/integration_tests/test_full_example.py

```diff
@@ -83,6 +97,29 @@ def template():
                 return_list=True,
             ),
         },
+        "Bzip2JsonMd": {
+            "description": mapped("json", "description"),
```

Comment:
Hitting the wrong source here. The bzip2 code isn't actually being tested yet.
```suggestion
            "description": mapped("bzip2json", "description"),
```

Reply:
Might be worth changing some of the queries, or the data inside of the zipped file to avoid such mistakes.